nixos/modules/services/security/fprintd.nix

```diff
@@ -25,6 +25,16 @@ in
         '';
       };
 
+      package = mkOption {
+        type        = types.package;
```

Comment:
Please don't align the `=` vertically. It's not a widely used style or maintained.

Reply:
fixed